src/Compilers/Core/CodeAnalysisTest/Diagnostics/OperationTestAnalyzer.cs

```diff
@@ -1777,7 +1763,7 @@ public override void Visit(IOperation operation)
                     }
                     if (operation.Kind == OperationKind.Argument)
                     {
-                        if (((IArgument)operation).ArgumentKind == ArgumentKind.ParamArray)
+                        if (((IArgument)operation).Parameter.IsParams)
```

Comment:
Not sure about intent here. ParameterSymbol.IsParams doesn't guarantee that the method was used in expanded form.